_posts/for-developers/apps/vocabularies/2019-01-01-05_common.md

```diff
@@ -161,10 +161,10 @@ This vocabulary helps describing copyright licenses.
 
 ```turtle
 # The Obelisk ontology is available under the CC 4.0 license...
-<http://w3id.org/obelisk/ns/> cc:license <http://creativecommons.org/licenses/by/4.0/>.
+<http://w3id.org/obelisk/ns/> cc:license <https://creativecommons.org/licenses/by/4.0/>.
```

Comment:
We shouldn't change that, I think. There is a Design Issue on it: https://www.w3.org/DesignIssues/Security-NotTheS.html
and it seems the CC server does the right thing when upgrading.